src/Adapter/Module/ModuleDataProvider.php

```diff
@@ -25,20 +25,108 @@
  */
 namespace PrestaShop\PrestaShop\Adapter\Module;
 
-/**
- * This class will provide data from DB / ORM about Module
- */
+use PrestaShop\PrestaShop\Adapter\Shop\Context;
+
 class ModuleDataProvider
 {
+    public function findByName($name)
```

Comment:
this class should be called ModuleRepository regarding the consistency with Theme implementation.